Google.PowerShell/BigQuery/BqTable.cs

```diff
@@ -464,19 +475,19 @@ protected override void ProcessRecord()
                         {
                             response = request.Execute();
                         }
-                        else if (ShouldContinue(
+                        else if (yesToAll || ShouldContinue(
```

Comment:
I am pretty sure you can skip the empty check once yesToAll is set. Treat yesToAll like `Force`, and move it to that `if`.